doc/Config_Editing_Guide.md

```diff
@@ -0,0 +1,166 @@
+# About
+I reccomend viewing this document with the extension Markdown Preview Enhanced (https://marketplace.visualstudio.com/items?itemName=shd101wyy.markdown-preview-enhanced). Once installed to VS Code you can see a html preview by clicking the pane on the right side of VS Code that looks like an open book with a magnifying glass.
+
+# Config Editing Tutorial 1
+This tutorial will take you through much of what is used in reskins and model swaps
+
+### Defining Your Object
+```c++
+/*The below code line defines our new object's (helmet's) internal name and tells ARMA via inheritance ':' that it is taking on the settings of our standard marine helm.
+This is done here because want the armor values of our marine helmet.*/
+class 34thPRC_ArmourStandard_Commando_Helmet_Urban : 34thPRC_ArmourStandard_Marines2_CH252_Urban
+```
+### Basic Object Detials
+```c++
+{
+	displayName="[34th] Commando Helmet Urban"; //This defines the in game display name of the gear
+	author="Yandere"; //Defines who made it
+	model="MA_Armor\data\Helmets\Commando\Commando_Helmet.p3d"; //This is part 1 of model swapping, both defining the model here and later on with uniformModel is required for most appearance swaps. URLs should be encased with "" and the line must end with a semi colon ;

```

Comment:
I feel like model swapping should be explained later in its own section after a basic texture swap example.